packages/codegen/src/index.test.ts

```diff
@@ -60,23 +60,28 @@ describe("generateSource", () => {
     expect(src).toContain(`export type Option = ("one" | "two" | "three")[];`);
   });
 
-  it("should handle properties both inside and outside of allOf", async () => {
+  it(async () => {
```

Comment:
Just nested these to deduplicate setup.

Reply:
Is it a common patter to nest `it` calls? Would have expected to use `describe` + `beforeEach|beforeAll` here.